### DISCUSSION TITLE
'Save pins' button doesn't work

### DISCUSSION THREAD

**Discussion by joonyle99** [community/community#84194] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

'Save pins' button under 'customize your pins' doesn't work in my GitHub profile.
I just clicked gist checkbox and 'Save pins' button because I want to pin a gist, but can't.

Also, I'm new to GitHub discussions, so please help me out if I'm missing something.

![Problem](https://github.com/community/community/assets/67359781/6e9d975c-6391-4727-bbcc-fcff24fb08b7)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @joonyle99! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
